src/python/bcc/__init__.py

```diff
@@ -1093,7 +1093,12 @@ def cleanup(self):
             # non-string keys here include the perf_events reader
             if isinstance(k, str):
```

Comment:
I used "return 1" twice, in dcsnoop and execsnoop, I think only by accident. I don't think those tools need it.

```
# grep "return 1" *.py
dcsnoop.py:    return 1;
execsnoop.py:    return 1;
tcptracer.py:  return 1;
tcptracer.py:  return 1;
```